examples/armv4t/gdb/host_io.rs

```diff
@@ -0,0 +1,35 @@
+use gdbstub::target;
+
+use crate::emu::Emu;
+
+impl target::ext::host_io::HostIo for Emu {
+    fn open(&self, filename: &[u8], _flags: u64, _mode: u64) -> i64 {
+        if filename == b"/proc/1/maps" {
```

Comment:
could you provide some context into what the GDB client is trying to do when accessing /proc/1/maps?

In any case, I'm not sure if this is something I'd like to include in the armv4t example, since the idea behind this example is that the GDB client is connecting to a simple bare-metal system, rather than one backed by a full-blown unix environment.

Reply:
Also, if I'm not mistaken, this seems to be replicating the functionality provided by:

https://github.com/daniel5151/gdbstub/pull/54

Reply:
I provide that command in the `GDB output` section of this PR.

This is not the same with #54.

I want to replace this implementation with reading ELF file after `exec-file` is implemented, so all the things is done by `GDB` automatically. ([previous link](https://github.com/daniel5151/gdbstub/commit/250568f547c43f7bbd7afdf8243592748df1d175#diff-0e0920511f8f247f9c2458696ec1bac7f523408824b393418a9ecd44469ba8a0)) That's why I squash the implementation of `exec-file` and Host I/O in one PR at the first time.

Reply:
As mentioned earlier, you can use the `remote get/put/delete` commands to validate this handler a bit more directly.